packages/nimble-blazor/NimbleBlazor/Components/NimbleTableColumnText.razor.cs

```diff
@@ -5,6 +5,12 @@ namespace NimbleBlazor;
 
 public partial class NimbleTableColumnText : ComponentBase
 {
+    /// <summary>
+    /// The ID of the column of a <see cref="NimbleTable{TData}"/>
+    /// </summary>
+    [Parameter]
+    public string? ColumnId { get; set; }
```

Comment:
I believe the appropriate thing to do here is create a new base class for our columns to derive from (i.e. `NimbleTableColumn`), and have this parameter exist there.

Reply:
done